cmd/notification.go

```diff
@@ -304,11 +302,11 @@ func (sys *NotificationSys) SignalService(sig serviceSignal) []NotificationPeerE
 // ServerInfo - calls ServerInfo RPC call on all peers.
 func (sys *NotificationSys) ServerInfo(ctx context.Context) []ServerInfo {
 	var idx = 0
-	serverInfo := make([]ServerInfo, len(sys.peerRPCClientMap))
+	serverInfo := make([]ServerInfo, len(sys.peerHosts))
 	var wg sync.WaitGroup
-	for addr, client := range sys.peerRPCClientMap {
+	for _, addr := range sys.peerHosts {
```

Comment:
We don't need idx anymore it was used with map previously, since now we have peerHosts as slice we can simply do 

```
for idx, addr := range sys.peerHosts {
```